robot/Cumulus/resources/NPSP.py

```diff
@@ -329,13 +314,13 @@ def verify_header(self,value):
         locator= npsp_lex_locators['header'].format(value)
         self.selenium.page_should_contain_element(locator)    
         
-    def Verify_affiliated_contact(self,list_name,first_name,last_name, y):   
-        """Validates if the affiliated contacts have the added contact details enter Y for positive case and N for negative case"""
-        name = first_name + ' ' + last_name
+    def verify_related_list(self,list_name,status,name):   
+        """If status is 'contains' then the specified related list should contain name
+                        'does not contain' then the specified related list should not contain name"""
         locator = self.salesforce.get_locator('record.related.link', list_name, name)
-        if y.upper()=="Y":
+        if status=="contains":
```

Comment:
status.lower() == "contains", maybe a good idea to avoid confusion

Reply:
thats why i specified in documentation to pass either 'contains' or 'does not contain'

Reply:
Yes true, just a safe practice to avoid human errors. Not a must. Just a suggestion.